jekyll/_data/sidenav.yml

```diff
@@ -134,6 +134,8 @@ en:
           link: 2.0/runner-api/
         - name: Runner FAQs
           link: 2.0/runner-faqs/
+        - name: Runner in Server
+          link: 2.0/runner-server/
```

Comment:
```suggestion
        - name: Runner in Server
          link: 2.0/runner-server/
        - name: Runner FAQs
          link: 2.0/runner-faqs/
```

Reply:
Looks good -- just rearranged the sidebar links so FAQ is at the bottom. This is fine to be merged now that Server 3 docs are public.